core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -78,7 +78,8 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('contao.csrf_token_name', $config['csrf_token_name']);
         $container->setParameter('contao.pretty_error_screens', $config['pretty_error_screens']);
         $container->setParameter('contao.error_level', $config['error_level']);
-        $container->setParameter('contao.locales', $config['locales']);
+        $container->setParameter('contao.locales', $config['locales'] ?: $config['backend_locales']); // Deprecated
```

Comment:
```suggestion
        $container->setParameter('contao.locales', $config['locales'] ?: $config['backend_locales']); // Backwards compatibility
```

I am always using the comment "backwards compatibility" for BC stuff, so we can find it more easily once we are removing it.